src/commands/setup.js

```diff
@@ -44,7 +45,28 @@ async function getUserInfoFromApi() {
       };
       return acc;
     }, {});
-  certSpinner.succeed(`Your certifications:\n${JSON.stringify(config.certs, null, 2)}`);
+  certSpinner.succeed('Your certifications:');
+
+  // Create a new table for certifications
+  const certsDetails = new Table({
+    head: [
+      {hAlign: 'center', content: 'id'},
+      {hAlign: 'left', content: 'project name'},
+      {hAlign: 'center', content: 'price'}],
+    colWidths: [5, 40, 7],
+  });
+
+  certifications.body
+    .filter(cert => cert.status === 'certified')
+    .forEach((cert) => {
+      certsDetails.push([
+        {hAlign: 'center', content: cert.project.id},
+        {hAlign: 'left', content: cert.project.name},
+        {hAlign: 'center', content: cert.project.price},
+      ]);
+    });
+  console.log(`${certsDetails.toString()}\n`);
```

Comment:
You could move the `spinner.succeed(string)` call here and incorporate the `certDetails` table.